.github/actions/is-cache-disabled/action.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
🔧  As you mentioned we should re-use the implementation of https://github.com/camunda/infra-global-github-actions/pull/312 when we move forward with this approach.

Reply:
I approved https://github.com/camunda/infra-global-github-actions/pull/312 so you are free to merge it and apply the refactoring